set-web/src/main/java/eu/europeana/set/web/service/impl/BaseUserSetServiceImpl.java

```diff
@@ -868,6 +867,10 @@ PersistentUserSet updateUserSetForPublish(PersistentUserSet userSet,
       userSet.setCreator(creator);
     }
     userSet.setVisibility(VisibilityTypes.PUBLISHED.getJsonValue());
+    if(issued==null) {
+      issued = new Date();
+    }
+    userSet.setIssued(issued);
```

Comment:
the specs were changed, this needs to be always executed, therefore the check from line 858 must be removed